infra/bridge/src/controllers/identity.js

```diff
@@ -0,0 +1,167 @@
+'use strict'
+
+const express = require('express')
+const get = require('lodash/get')
+
+const router = express.Router()
+const { GrowthEvent } = require('@origin/growth-event/src/resources/event')
+const {
+  loadIdentityAttestationsMetadata,
+  loadIdentityAddresses,
+  recordGrowthProfileEvent,
+  recordGrowthAttestationEvents
+} = require('@origin/identity/src/utils')
+const Identity = require('@origin/identity/src/models').Identity
+const validator = require('@origin/validator')
+
+const { pinIdentityToIpfs, postToEmailWebhook } = require('../utils/identity')
+const {
+  identityReadVerify,
+  identityWriteVerify
+} = require('../utils/validation')
+
+const logger = require('../logger')
+
+/**
+ * Loads the identity associated with an eth address.
+ * Return status:
+ *  - 200 if an identity was found in the database.
+ *  - 204 if no identity found.
+ *  - 500 if an identity row was found but with empty data in the database.
+ *
+ *  @args {string} req.ethAddress Eth address of the user.
+ *  @returns {{identity: Object, ipfsHash: string}}
+ */
+router.get('/', identityReadVerify, async (req, res) => {
+  const ethAddress = req.query.ethAddress.toLowerCase()
+  logger.debug(`identity/read called for addr ${ethAddress}`)
+
+  // Lookup the identity associated with the eth address.
+  const identity = await Identity.findOne({ where: { ethAddress } })
```

Comment:
Just asking for a clarification, is `ethAddress` always the owner address? If yes, shouldn't we check if it is proxy and find its owner?

Reply:
I plan on having the graphQL code always passing the owner's address... but you are right, better to make this more resilient by checking for proxy address here. Good thinking... will do !

Reply:
Done